src/Views/BucketPanel/EntryDetail.tsx

```diff
@@ -181,6 +285,153 @@ export default function EntryDetail(props: Readonly<Props>) {
           onRemoved={() => history.push(`/buckets/${bucketName}`)}
         />
       )}
+
+      <Modal
+        title="Upload File"
+        open={isUploadModalVisible}
+        onCancel={() => {
+          setIsUploadModalVisible(false);
+          setUploadError("");
+          uploadForm.resetFields();
+          setUploadFile(null);
+          setLabels([]);
+        }}
+        footer={null}
+        width={600}
+      >
+        <Form
```

Comment:
I just figured out that we need the entry field. The problem is that a new bucket has no entries and we can't create an empty entry.  So the form must have the input field: 
1) When we open the dialog form from the item view, it's disabled and holds the current item name. 
2) When we open the dialog form from the bucket view (we need to place it there), then a user must select the item name from the drop-down list or write a new one.